packages/web-components/docs/api-report.md

```diff
@@ -7,16 +7,77 @@
 import { CSSDesignToken } from '@microsoft/fast-foundation';
 import { ElementStyles } from '@microsoft/fast-element';
 import { ElementViewTemplate } from '@microsoft/fast-element';
+import { FASTAccordion } from '@microsoft/fast-foundation';

```

Comment:
Could you please rebase and run `yarn buildto web-components` from the repo root? There is no reason spec should contain api.md changes.

Reply:
Apologies. I'll get this right someday. Updated now.